src/AudioIOBase.cpp

```diff
@@ -26,7 +26,7 @@ Paul Licameli split from AudioIO.cpp
 #endif
 
 #ifdef EXPERIMENTAL_MIDI_OUT
-#include "../lib-src/portmidi/pm_common/portmidi.h"
```

Comment:
Wherever there is `#include "../lib-src/..."` there is something wrong.

I count 14 such places in the source tree now (+1 that is commented out).  This removes four of those.

I'm glad to see that.  Will we see removal of the rest of them?

Reply:
Will surely do :)

Reply:
Wait now, there is one inclusion of portmidi.h from NoteTrack.cpp that was not removed in this commit.  Did the next commit do it?